compiler/src/dotty/tools/dotc/core/tasty/TastyPrinter.scala

```diff
@@ -21,41 +23,56 @@ class TastyPrinter(bytes: Array[Byte])(implicit ctx: Context) {
   def printNames(): Unit =
     for ((name, idx) <- nameAtRef.contents.zipWithIndex) {
       val index = nameColor("%4d".format(idx))
-      println(index + ": " + nameToString(name))
+      sb.append(index + ": " + nameToString(name) + "\n")
```

Comment:
`sb.append(index).append(": ").append(nameToString(name)).append("\n")` is more efficient.